Makefile.am

```diff
@@ -87,9 +75,7 @@ libsass_la_SOURCES = \
 	utf8_string.cpp utf8_string.hpp \
 	util.cpp util.hpp
 
-libsass_la_CFLAGS = $(AM_CFLAGS)
-libsass_la_CXXFLAGS = $(AM_CXXFLAGS)
-libsass_la_LDFLAGS = $(AM_LDFLAGS) -no-undefined -version-info 0:9:0
+libsass_la_LDFLAGS = -no-undefined -version-info 0:9:0
```

Comment:
Correct. I need to see what we can do with test coverage.

Unfortunately I think we might want to pass `-O2` to the linker too.